src/webui/www/private/scripts/contextmenu.js

```diff
@@ -413,13 +433,45 @@ var TorrentsTableContextMenu = new Class({
             }
             categoryList.appendChild(el);
         });
+    },
+
+    updateTagsSubMenu: function(tagList) {
+        const contextTagList = $('contextTagList');
+        while (contextTagList.firstChild)
+            contextTagList.removeChild(contextTagList.firstChild);
+
+        contextTagList.appendChild(new Element('li', {
+            html: '<a href="javascript:torrentAddTagsFN();"><img src="images/qbt-theme/list-add.svg" alt="QBT_TR(Add...)QBT_TR[CONTEXT=TransferListWidget]"/> QBT_TR(Add...)QBT_TR[CONTEXT=TransferListWidget]</a>'
```

Comment:
Please don't produce such long lines. Split them all.

Reply:
Should I use concatenation or newline escaping to construct multiline literals?

Reply:
I would use concatenation. @Chocobo1?

Reply:
If using `\` doesn't break the indention so much... I'm fine with both way.